app/controllers/conference_attendances_controller.rb

```diff
@@ -1,19 +1,18 @@
 class ConferenceAttendancesController < ApplicationController
 
   def update
-    @conference_attendance = ConferenceAttendance.find(attendance_params[:id])
-    @team = Team.find(attendance_params[:team_id])
+    @conference_attendance = ConferenceAttendance.find(params[:id])
+    @team = @conference_attendance.team
 
     if @conference_attendance.update_attributes(attendance_params)
       redirect_to @team, notice: 'Your attendance was successfully recorded.'
```

Comment:
You could still skip line 5 and just use `@conference_attendance.team` here (if I'm not missing any other uses).
That way the only assigned @-variable would be `@ conference_attendance`, signaling that this is the object that we are dealing with in this action. (But this is really a small code style thing.)